R/read_alevin.R

```diff
@@ -4,16 +4,14 @@
 #' @param mtx_format Logical indicating if input data is in matrix market format.
 #'   Default is FALSE.
 #' @param intron_mode Logical indicating if the files included alignment to intronic regions.
+#'   If TRUE, the main "counts" assay will contain counts aligned to introns and an additional "spliced"
+#'   assay will contain counts aligned to spliced cDNA only.
```

Comment:
I think we should probably rename this... Not quite sure how. 

I was going to say `spliced_only` or something like that, but in the case where there are no unspliced data, I'm not quite sure what we are planning to do.

Maybe `include_unspliced` since that is what you use later to store this value?